content/docs/command-reference/run.md

```diff
@@ -7,11 +7,12 @@ command.
 
 ```usage
 usage: dvc run [-h] [-q | -v] [-n <name>] [--no-exec] [--no-run-cache]
-               [--no-commit] [-d <path>] [-o <filename>] [-O <filename>]
-               [-p [<filename>:]<params_list>] [-m <path>] [-M <path>]
-               [--plots <path>] [--plots-no-cache <path>] [--live <path>]
-               [--live-no-cache <path>] [--live-no-html]
-               [-w <path>] [-f] [--outs-persist <filename>]
+               [--no-commit] [-d <path>] [-o <filename>]
+               [-O <filename>] [-p [<filename>:]<params_list>]
+               [-m <path>] [-M <path>] [--plots <path>]
+               [--plots-no-cache <path>] [--live <path>]
+               [--live-no-cache <path>] [--live-no-html] [-w <path>]
+               [-f] [--outs-persist <filename>]
```

Comment:
OK this is a wrapping that respects the order from the help output and doesn't scroll horizontally at min width on Windows (up to 72 chars), but it seems like on Mac it does (68 chars) 🤷🏼 

![image](https://user-images.githubusercontent.com/1477535/157403765-529f0af3-7de5-444d-b52b-48a76b3d1959.png)